mzLib/FlashLFQ/IsotopicEnvelope.cs

```diff
@@ -18,6 +19,14 @@ public IsotopicEnvelope(IndexedMassSpectralPeak monoisotopicPeak, int chargeStat
             Intensity = intensity / chargeState;
         }
 
+        public IsotopicEnvelope(IndexedMassSpectralPeak monoisotopicPeak, int chargeState, double intensity, double pearsonCorrelation)
```

Comment:
why do you need a second constructor? If backwards compatability is an issue, could you make pearson correlation have a default in the original constructor?

Reply:
Good question. Pearson correlation isn't actually used, it was something I had implemented and abandoned. I deleted all references